scripts/release.sh

```diff
@@ -7,6 +7,9 @@ NAME=$(node --eval "console.log(require('./package.json').name);")
 # build and test
 npm test || exit 1
 
+# update integrity
+npm run integrity
```

Comment:
This runs BEFORE the NPM publish so it won't get the integrity data for the latest version. I think this has to run AFTER `npm publish` on line 37 as the last thing in the scripts.

Reply:
Thanks Patrick, this is a good discussion: If we move it to AFTER the `npm publish`, then we'll also have to make sure we do a final additional commit on the master branch and then it would be available at `https://github.com/Esri/esri-leaflet/blob/master/siteData.json` - I'll update `release.sh` to do that. 

(I'm also a bit concerned about the timing - not sure if the NPM API will immediately start reporting the new version number or if it takes a few seconds because of CDN caching  - I guess we'll just have to try and see though)